### PR TITLE
SCRUM-4300 add expression ribbon fields

### PR DESCRIPTION
**Pull request by adamgibs**

### CODE REVIEW

File: model/schema/expression.yaml [hunk 1]
```diff
@@ -511,6 +513,18 @@ slots:
     range: GOTerm
     multivalued: false
 
+  cellular_component_ribbon_term:
+    description: >-
+      The primary cellular component slim term referred to within the context of an AnatomicalSite object and used by the expression ribbon.
+    domain: AnatomicalSite
```
Comment (oblodgett) [inline model/schema/expression.yaml:519]: @adamgibs shouldn't this be a GOTerm not an AnatomicalSite?

Comment (markquintontulloch) [reply model/schema/expression.yaml:519]: `domain: AnatomicalSite` is correct but it does need `range: GOTerm` too

Comment (adamgibs) [reply model/schema/expression.yaml:519]: Okay, I've just updated it.

**Review by markquintontulloch — APPROVED** [review]

File: model/schema/expression.yaml [hunk 2]
```diff
@@ -511,6 +513,20 @@ slots:
...
+    range: GOTerm
+    multivalued: false
+
+  cellular_component_other:
+    description: >-
+      Is set to true when no cellular_component_ribbon_term is available
+    domain: AnatomicalSite
+    range: boolean
```
Comment (oblodgett) [inline model/schema/expression.yaml:527]: Is this range correct?

Comment (adamgibs) [reply model/schema/expression.yaml:527]: It matches other boolean values in the model like [here](https://github.dev/alliance-genome/agr_curation_schema/blob/b25a1dfb8f019d69d571664ec4b6e10dd877b8f9/model/schema/allele.yaml#L1024) so I think so.

**Pull request merged**